container/root/tests/php-fpm/base.goss.yaml

```diff
@@ -149,3 +151,42 @@ command:
   php -n -d zend_extension=xdebug.so --ri xdebug:
     exit-status: 0
     stderr: ['!/./']
+
+file:
+  {{ .Env.CONF_FPMPOOL }}:
```

Comment:
why didn't we notice this earlier... :(

Reply:
This is so much better, definitely missed this the first time 😦 

Good find